pkg/manager/impl/execution_manager.go

```diff
@@ -1153,11 +1153,16 @@ func (m *ExecutionManager) ListExecutions(
 		return nil, errors.NewFlyteAdminErrorf(codes.InvalidArgument, "invalid pagination token %s for ListExecutions",
 			request.Token)
 	}
+	joinTableEntities := make(map[common.Entity]bool)
+	for _, filter := range filters {
```

Comment:
how is this filter specified? One thing I have been struggling with is filters documentation. Can you please point me to one or an example?

Reply:
docs here: https://docs.lyft.net/eng/flytedocs/contributor/components/admin.html#making-requests-to-flyteadmin let me know if i can move this somewhere more discoverable

Reply:
Ohh my bad, should always know Katrina has docs. Stand corrected